dcos/dcos-ui
# [Spike] DCOS-51565: use default styling for disabled banner

Use default styling for the container form section disabled banner.

Closes DCOS-51565

## Testing

- Open jobs form
- Make sure Command Only option is selected in General tab
- Go to Container tab, disabled banner should have default styling (grey background)

## Trade-offs

N/A

## Dependencies

N/A

## Screenshots

Before:
<img width="771" alt="Screen Shot 2019-04-11 at 10 22 09 AM" src="https://user-images.githubusercontent.com/19582796/55977455-b2923000-5c43-11e9-92ff-9216f6d39d30.png">


After:
<img width="775" alt="Screen Shot 2019-04-11 at 10 21 26 AM" src="https://user-images.githubusercontent.com/19582796/55977416-9abaac00-5c43-11e9-8a7a-1e89985627ea.png">
